packages/create-llama/templates/types/simple/fastapi/README-template.md

```diff
@@ -9,9 +9,10 @@ poetry install
 poetry shell
 ```
 
-Second, run the development server:
+Second, export your API key. Then run the development server:
 
 ```
+export $(cat .env)
```

Comment:
It shouldn't be required as the Python backend is calling:

https://github.com/run-llama/LlamaIndexTS/blob/b64716d3f7e41e0b8db7421b7150cafd13026843/packages/create-llama/templates/types/simple/fastapi/main.py#L9

@dantheman0207 Is there something special in your environment that prevents the .env from being loaded? Maybe you already have another API key set?

Reply:
I don't think so, fresh install and set the api key via prompt. I don't believe I did have anything in my env but if I did it would have been the same API key and same variable name. When I added it here it failed but I no longer have the exact error. I then manually exported the env file and it failed with a different error (HTTP 401 as OpenAI needed me to verify my phone number). I updated to a different API key but it didn't update until I manually exported again. Strange as I wondered why it wasn't just loaded with dotenv but I see now it is. If I `unset OPENAI_API_KEY` and put the empty string in my .env it fails. But then when I update the .env to again have the same key it still fails with error "No API key found for OpenAI."

Reply:
> Author

I also have the same issue. I did not modify anything after the installation completed except the OPENAI_API_KEY.